pytorch_blade/tests/torch-disc-pdll/tests/fake_quant.mlir

```diff
@@ -0,0 +1,18 @@
+// RUN: torch-disc-pdll --payload-input %s --pdl-input %p/fake_quant.pdll  | FileCheck %s
+
+// CHECK-LABEL:  func.func @torch_blade_custom_fake_quant
+func.func @torch_blade_custom_fake_quant(%arg0: !torch.tensor, %arg1: !torch.tensor, %arg2: !torch.tensor) -> !torch.tensor {
```

Comment:
One more thing to discuss is where this pass should be placed since it will affects the content of the test case (e.g. use torch.tensor or torch.vtensor and so on)

Reply:
It will be implemented in pr merging torch pdll utils.